src/libCom/osi/os/WIN32/osdEvent.c

```diff
@@ -93,16 +93,23 @@ epicsShareFunc epicsEventStatus epicsEventWaitWithTimeout (
     epicsEventId pSem, double timeOut )
 { 
     static const unsigned nSec100PerSec = 10000000u;
```

Comment:
Probably unrelated, but I happened to notice that this constant is 10 million (7 zeros).

Also, Is the name `nSec100` meant to convey `nSec / 100`?

Reply:
`nSec100` is for the number of `100ns` intervals, which is the unit used by waitable timers, probably because windows `FILETIME` uses this. That then gives 10^7 of these intervals per second.

Reply:
Ok, I see.  I'm not asking for a change, just noting that the constant name isn't so helpful to me.  Were it me, I would skip the named constant and instead add a comment.  eg. for `timeOut >= INFINITE / 1000u` something to the effect that the high limit is `INFINITE` milliseconds, which originated from historical use of `WaitForSingleObject()`.

Reply:
Michael said he wasn't asking for change but I think it may be warranted (since the use of 100ns is confusing) – could you introduce a name for that interval, and adjust the other names to match? We may re-use the term "tick" too much although I wouldn't object if you want to use it, you already called 100ns an "interval" (ival, intv, intvl?), but anything else you come up with might work instead. The easier we can make the code for humans to read the less likelihood of bugs being introduced and the faster it is to change it later.

Reply:
thanks - I chose "ival"